doc/target.rst

```diff
@@ -450,3 +450,112 @@ Target
     Returns the path to the extracted contents. In case of files (gzip and
     bzip2), the path to the decompressed file is returned; for archives, the
     path to the directory with the archive's contents is returned.
+
+
+Android Target
+---------------
+
+.. class:: AndroidTarget(connection_settings=None, platform=None, working_directory=None, executables_directory=None, connect=True, modules=None, load_default_modules=True, shell_prompt=DEFAULT_SHELL_PROMPT, conn_cls=AdbConnection, package_data_directory="/data/data")
+
+    :class:`AndroidTarget` is a subclass of :class:`Target` with additional features specific to a device running Android.
+
+    :param conn_cls: This is the type of connection that will be used to communicate
+        with the device. The majority of Android devices will use an ADB connection.
+
+    :param package_data_directory: This is the location of the data stored for
+        installed Android packages on the device.
+
+.. method:: Target.set_rotation(rotation)
+
+   Sets the screen orientation of the device as an integer value
+   representing the screen orientations. ``Natural : 0``, ``Rotated Left: 1``,
+   ``Inverted : 2`` and ``Rotated Right : 3``.
```

Comment:
Should be more explicit about what `param` values should/can be. E.g. can i specify the text form just the number? Also only  use monospace for literals and identifiers.